apps/crn-frontend/package.json

```diff
@@ -41,7 +41,7 @@
     "react-error-boundary": "3.1.4",
     "react-router-dom": "5.3.4",
     "react-router-last-location": "2.0.1",
-    "recoil": "patch:recoil@^0.1.2#../../patches/recoil.patch",
+    "recoil": "patch:recoil@0.7.0#../../patches/recoil.patch",
```

Comment:
can we not just have a normal version here?

Reply:
No

Reply:
I take it back. Upgraded recoil again and tried to patch again and the type change was already there 🎉